pyquil/api/_compiler.py

```diff
@@ -160,7 +163,7 @@ def __init__(self,
                  device: AbstractDevice,
                  timeout: int = 10,
                  name: Optional[str] = None,
-                 config: Optional[PyquilConfig] = None) -> None:
+                 session: Optional[ForestSession] = None) -> None:
```

Comment:
lets just make this a keyword only arg by tossing `/` before it -- makes it much easier to refactor in the future if such is necessary

Reply:
Assuming you meant `*`, good idea, fixed

Reply:
yeah -- the `/` is for positional-only haha